src/element_ref/element.rs

```diff
@@ -1,11 +1,11 @@
 use selectors::{Element, OpaqueElement};
```

Comment:
While in my previous commits I've disabled the automatic code formatting (for readability reasons), here I've let it run over my changes.

if you don't like this, I can simply revert this commit. Or I can run the linter on the rest of the `scraper` sources and finish the job ;-)